example/pubspec.yaml

```diff
@@ -7,11 +7,11 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
-  flame: ^1.2.0
+  flame: ^1.3.0
   flame_behaviors:
     path: ../
   flutter:
     sdk: flutter
 
 dev_dependencies:
-  very_good_analysis: ^3.0.0
+  very_good_analysis: ^3.0.1
```

Comment:
did we update the actual version used in the `analysis_options.yaml`?

Reply:
I meant https://github.com/VeryGoodOpenSource/flame_behaviors/blob/feat/update-version-constraitns/analysis_options.yaml haha looks like it's still using 3.0.0